### ISSUE TITLE
new jsrsasign.KJUR.asn1.ocsp.OCSPRequest cannot handle P384

### ISSUE THREAD

**Issue by augjoh** [kjur/jsrsasign#500]
When trying to generate an OCSPRequest for a P384 signed CA certificate, jsrsasign throws the following error:

```
TypeError: Cannot read property 'length' of null
    at Object.parse (/usr/home/node/node_modules/jsrsasign/lib/jsrsasign.js:123:23)
    at KJUR.crypto.MessageDigest.updateHex (/usr/home/node/node_modules/jsrsasign/lib/jsrsasign.js:12668:42)
    at KJUR.crypto.MessageDigest.digestHex (/usr/home/node/node_modules/jsrsasign/lib/jsrsasign.js:12680:22)
    at hashHex (/usr/home/node/node_modules/jsrsasign/lib/jsrsasign.js:12569:18)
    at KJUR.asn1.ocsp.CertID.setByCert (/usr/home/node/node_modules/jsrsasign/lib/jsrsasign.js:11495:17)
    at new KJUR.asn1.ocsp.CertID (/usr/home/node/node_modules/jsrsasign/lib/jsrsasign.js:11517:18)
    at new KJUR.asn1.ocsp.Request (/usr/home/node/node_modules/jsrsasign/lib/jsrsasign.js:11604:17)
    at KJUR.asn1.ocsp.TBSRequest.setRequestListByParam (/usr/home/node/node_modules/jsrsasign/lib/jsrsasign.js:11622:21)
    at new KJUR.asn1.ocsp.TBSRequest (/usr/home/node/node_modules/jsrsasign/lib/jsrsasign.js:11650:18)
    at new KJUR.asn1.ocsp.OCSPRequest (/usr/home/node/node_modules/jsrsasign/lib/jsrsasign.js:11681:21)
```

Sample code:
```
> cat OCSPRequest.js
#!/usr/bin/env node

const jsrsasign = require('jsrsasign');
const fs = require('fs');

const caPem = '-----BEGIN CERTIFICATE-----' +
'MIIDmzCCAyKgAwIBAgIVAL6At26+LYDhyCcuZ/MbtNt+j+c5MAoGCCqGSM49BAMD' +
'MFkxCzAJBgNVBAYTAkRFMRIwEAYDVQQHDAlCaWVsZWZlbGQxEDAOBgNVBAoMB015' +
'LkNvcnAxFTATBgNVBAsMDFBLSSBTZXJ2aWNlczENMAsGA1UEAwwEUm9vdDAeFw0y' +
'MTA2MTkxNDU1NDhaFw0yMzEyMjIxNDU1NDhaMGMxCzAJBgNVBAYTAkRFMRIwEAYD' +
'VQQHDAlCaWVsZWZlbGQxEDAOBgNVBAoMB015LkNvcnAxFTATBgNVBAsMDFBLSSBT' +
'ZXJ2aWNlczEXMBUGA1UEAwwOU3ViIENBIDIwMjEgRzEwWTATBgcqhkjOPQIBBggq' +
'hkjOPQMBBwNCAAQvd5/f6z6YOK85KLC37SwzA0xgjntQoHom0G+5uofIskb0+MKn' +
'pLZBzYHUsztBgOSGG/qHVNn95i7TU9qH6lQ6o4IBuzCCAbcwEgYDVR0TAQH/BAgw' +
'BgEB/wIBADAOBgNVHQ8BAf8EBAMCAYYwTwYDVR0gBEgwRjAIBgZngQwBAgEwOgYC' +
'KQgwNDAyBggrBgEFBQcCARYmaHR0cHM6Ly9ub2RlanMucnotYnNkLm15LmNvcnAv' +
'Y3BzL1Jvb3QwegYIKwYBBQUHAQEEbjBsMGoGCCsGAQUFBzAChl5odHRwOi8vbm9k' +
'ZWpzLnJ6LWJzZC5teS5jb3JwL2Rvd25sb2FkL1Jvb3QvNjE4YzY5Mzg2NGFiYmZi' +
'ZTVmYzUyODI0MjA1MDZjOThhNmNhMmRiMi9jYS5jcnQuY2VyMGwGA1UdHwRlMGMw' +
'YaBfoF2GW2h0dHA6Ly9ub2RlanMucnotYnNkLm15LmNvcnAvZG93bmxvYWQvUm9v' +
'dC82MThjNjkzODY0YWJiZmJlNWZjNTI4MjQyMDUwNmM5OGE2Y2EyZGIyL2NybC5k' +
'ZXIwKwYDVR0jBCQwIoAg9AycIcTKexjLPwSXjrIfkH5ae5zM3gcEh8JD0qD5vWow' +
'KQYDVR0OBCIEIMK80iMp6LixgyWzZrdq/aOECREjtveACqMcgjMY0yBAMAoGCCqG' +
'SM49BAMDA2cAMGQCME2Z4Bz4I5rrvqdeGMIhDlnNs7UHudsf7GwWRX1JVqqPWA9e' +
'2kUZogzYooIIWE5UiQIwC6EuuTYWCUPeYY8WemAgpMXn9B1JpHCZPDXJy5noGdNl' +
'IQE0PDbLrUPE0Y2Vzf+/' +
'-----END CERTIFICATE-----';
const certPem = '-----BEGIN CERTIFICATE-----' +
'MIIDvTCCA2OgAwIBAgIVAPqxKxbizlUExMsw+ulVCZByiqniMAoGCCqGSM49BAMC' +
'MGMxCzAJBgNVBAYTAkRFMRIwEAYDVQQHDAlCaWVsZWZlbGQxEDAOBgNVBAoMB015' +
'LkNvcnAxFTATBgNVBAsMDFBLSSBTZXJ2aWNlczEXMBUGA1UEAwwOU3ViIENBIDIw' +
'MjEgRzEwHhcNMjEwODEyMDczMzEzWhcNMjIwODEyMDczMzEzWjAbMRkwFwYDVQQD' +
'FhBmZXdpQHBsYXR5bnVtLmNoMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEPnSS' +
'1Uslp/ZjAIqstuEt0r5c5LZ3UIcpYd0XdcFGXSj4qTXacuvSyBO2RqDoKBofUjrf' +
'ZJgV+ig5dOMtdBD2q6OCAjowggI2MAkGA1UdEwQCMAAwDgYDVR0PAQH/BAQDAgOI' +
'MBYGA1UdJQEB/wQMMAoGCCsGAQUFBwMEMBsGA1UdEQQUMBKBEGZld2lAcGxhdHlu' +
'dW0uY2gwgdgGCCsGAQUFBwEBBIHLMIHIMFsGCCsGAQUFBzABhk9odHRwOi8vbm9k' +
'ZWpzLnJ6LWJzZC5teS5jb3JwL29jc3AvU3ViL2JlODBiNzZlYmUyZDgwZTFjODI3' +
'MmU2N2YzMWJiNGRiN2U4ZmU3MzkvMGkGCCsGAQUFBzAChl1odHRwOi8vbm9kZWpz' +
'LnJ6LWJzZC5teS5jb3JwL2Rvd25sb2FkL1N1Yi9iZTgwYjc2ZWJlMmQ4MGUxYzgy' +
'NzJlNjdmMzFiYjRkYjdlOGZlNzM5L2NhLmNydC5jZXIwawYDVR0fBGQwYjBgoF6g' +
'XIZaaHR0cDovL25vZGVqcy5yei1ic2QubXkuY29ycC9kb3dubG9hZC9TdWIvYmU4' +
'MGI3NmViZTJkODBlMWM4MjcyZTY3ZjMxYmI0ZGI3ZThmZTczOS9jcmwuZGVyMEQG' +
'A1UdIAQ9MDswOQYCKQgwMzAxBggrBgEFBQcCARYlaHR0cHM6Ly9ub2RlanMucnot' +
'YnNkLm15LmNvcnAvY3BzL1N1YjArBgNVHSMEJDAigCDCvNIjKei4sYMls2a3av2j' +
'hAkRI7b3gAqjHIIzGNMgQDApBgNVHQ4EIgQgdSKkZFRWsmZ/+F5/SrfQZmhTin1o' +
'4gdE+Gj6W3yMeB0wCgYIKoZIzj0EAwIDSAAwRQIgcvjkShbS2xKalMBgxwg2B+7B' +
'7LPEQAJz4GA3NVb7vmgCIQCgv711dhItJPJ3mqwf3w4HtbUXKX2dK+rvsozQEp4X' +
'Mw==' +
'-----END CERTIFICATE-----';

let x509 = new jsrsasign.X509();
x509.readCertPEM(caPem);
console.log(`${JSON.stringify(x509.getParam())}`);

let request = new jsrsasign.KJUR.asn1.ocsp.OCSPRequest({
    reqList: [{
        issuerCert: caPem,
        subjectCert: certPem,
        alg: 'sha256'
    }]
});
console.log(`${JSON.stringify(request)}`);
```

**Comment by kjur** [comment]
Try following:
```JavaScript
let request = new jsrsasign.KJUR.asn1.ocsp.OCSPRequest({
    reqList: [{
        issuerCert: caPem,
        subjectCert: certPem,
        alg: 'sha256'
    }]
});
console.log(request.getEncodedHex());
```

**Comment by augjoh** [comment]
@kjur This leads to the same error. I'm using the latest version from npmjs (10.3.0).

```
TypeError: Cannot read property 'length' of null
    at Object.parse (/usr/home/node/node_modules/jsrsasign/lib/jsrsasign.js:18:1595)
    at KJUR.crypto.MessageDigest.updateHex (/usr/home/node/node_modules/jsrsasign/lib/jsrsasign.js:234:4789)
    at KJUR.crypto.MessageDigest.digestHex (/usr/home/node/node_modules/jsrsasign/lib/jsrsasign.js:234:5007)
    at hashHex (/usr/home/node/node_modules/jsrsasign/lib/jsrsasign.js:234:2503)
    at KJUR.asn1.ocsp.CertID.setByCert (/usr/home/node/node_modules/jsrsasign/lib/jsrsasign.js:232:6952)
    at new KJUR.asn1.ocsp.CertID (/usr/home/node/node_modules/jsrsasign/lib/jsrsasign.js:232:7463)
    at new KJUR.asn1.ocsp.Request (/usr/home/node/node_modules/jsrsasign/lib/jsrsasign.js:232:8808)
    at KJUR.asn1.ocsp.TBSRequest.setRequestListByParam (/usr/home/node/node_modules/jsrsasign/lib/jsrsasign.js:232:9194)
    at new KJUR.asn1.ocsp.TBSRequest (/usr/home/node/node_modules/jsrsasign/lib/jsrsasign.js:232:9649)
    at new KJUR.asn1.ocsp.OCSPRequest (/usr/home/node/node_modules/jsrsasign/lib/jsrsasign.js:232:10244)
```

**Comment by kjur** [comment]
I've fixed this issue on the release 10.3.2 today.
